code/datums/looping_sounds/machinery_sounds.dm

```diff
@@ -17,6 +17,51 @@
 	volume = 50
 	range = 20
 
+/datum/looping_sound/river_loop
+	mid_sounds = list(
+		'sound/ambience/river/stream_1.ogg' = 1,
+		'sound/ambience/river/stream_2.ogg' = 1,
+		'sound/ambience/river/stream_3.ogg' = 1,
+	)
+	mid_length = 2.5 SECONDS
+	volume = 25
+	range = 14
+	falloff = 2
+	ambient_sound = TRUE
+
+/datum/looping_sound/wind_loop
+	mid_sounds = list('sound/effects/wind/wind_2_1.ogg' = 1, 'sound/effects/wind/wind_2_2.ogg' = 1, 'sound/effects/wind/wind_3_1.ogg' = 1, 'sound/effects/wind/wind_4_1.ogg' = 1, 'sound/effects/wind/wind_4_2.ogg' = 1, 'sound/effects/wind/wind_5_1.ogg' = 1)
+	mid_length = 30
+	volume = 40
+	range = 15
+
+/datum/looping_sound/water_res_loop
+	mid_sounds = list('sound/effects/wave1.ogg' = 1, 'sound/effects/wave2.ogg' = 1, 'sound/effects/wave3.ogg' = 1)
+	mid_length = 13
+	volume = 28
+	range = 7
+
+/datum/looping_sound/drip_loop
+	mid_sounds = list('sound/effects/drip1.ogg' = 1, 'sound/effects/drip2.ogg' = 1, 'sound/effects/drip3.ogg' = 1, 'sound/effects/drip4.ogg' = 1, 'sound/effects/drip5.ogg' = 1)
+	mid_length = 4
+	volume = 25
+	range = 7
+
+/datum/looping_sound/low_wind_loop
+	mid_sounds = list('sound/ambience/windambient.ogg' = 1)
+	mid_length = 18
+	volume = 28
+	range = 7
+
+/datum/looping_sound/indoor_hum_loop
+	mid_sounds = list('sound/ambience/ambienthum.ogg' = 1)
+	mid_length = 1
+	volume = 15
+	range = 8
+
+/datum/looping_sound/indoor_hum_loop/lowrange
+	range = 4
```

Comment:
these still need to be ambience if they're being used as ambience, unless they're explicitly like emitting from an atom or something.

Reply:
I'm not sure what's meant here, the only ones here that could be ambient are indoor hum and *maybe* the low_wind_loop.

Reply:
so drip, andwater_res is all good because they're just coming out of very specific sources. (but set them up with the ambient var like I did for river)

Both wind loops and indoor hums though are just all over the places in various areas, so really need to be area ambience.